tests/ci.js

```diff
@@ -142,39 +132,39 @@ assertTestFailure();
 
 shell.cd('../');
 
-shell.echo('### Testing elm-test on example-application-no-tests');
+shell.echo('\n### Testing elm-test on example-application-no-tests');
 
 shell.cd('example-application-no-tests');
 
 assertTestFailure();
 
 shell.cd('../');
 
-shell.echo('### Testing elm-test on example-package-no-core');
+shell.echo('\n### Testing elm-test on example-package-no-core');
 
 shell.cd('example-package-no-core');
 
-assertTestSuccess();
+assertTestErrored();
```

Comment:
I don’t know why this one was marked as success before, and why it passed before. Running `elm make` in example-package-no-core yields:

```
-- MISSING DEPENDENCY ------------------------------------------------- elm.json

I need to see an "elm/core" dependency your elm.json file. The default imports
of `List` and `Maybe` do not work without it.

If you modified your elm.json by hand, try to change it back! And if you are
having trouble getting back to a working elm.json, it may be easier to find a
working package and start fresh with their elm.json file.
```

So it makes sense that it should fail IMO.

Reply:
Test removed in https://github.com/rtfeldman/node-test-runner/pull/448